### ISSUE TITLE
CCGT Dessau not considered in generated energy and emissions

### ISSUE THREAD

**Issue by nesnoj** [windnode/WindNODE_ABW#161]
cf. single nb plots 4.11+5.3

**Comment by gplssm** [comment]
Quick update: CO2 emissions calculation works correctly. I verified number for CCGT power plants (excluding the one in Dessau). The trick was to use the correct efficiency, which is not the same as in parameter excel sheet. Gets recalculated in the model and as well in ex-post analysis

**Comment by gplssm** [comment]
I dived into a bit and found zero generation by GuD in Dessau. Even the extracted flow are zero

![grafik](https://user-images.githubusercontent.com/8255114/90406644-c5254080-e0a5-11ea-8c65-953333c4d490.png)

Hence, I would say regarding CO2 emission calculation everything is correct.

I'm unsure if it's plausible that the GuD in Dessau isn't generating electricity at all in a one-year simulation of `NEP_RE-`... @nesnoj?

**Comment by nesnoj** [comment]
Sounds reasonable but this must have changed with the new scenarios.. how's the district heating supplied then?

If everything is fine with the model, should we decommission the plant (at least 2050)...?

BTW: It's a pity the notebook lacks some plot on heat generation #163

**Comment by gplssm** [comment]
> Sounds reasonable but this must have changed with the new scenarios.. how's the district heating supplied then?

Does not really make sense to me that it changed with the new scenarios. If GuD Dessau was used in old NEP2035_* scenarios, with lesser RE generation in new scenarios it should be less dispatched.
I quickly check an exemplary `.lp` file for any odd parameters. Didn't find any.

The heat demand must the be supplied by the gas boiler entirely. This seems plausible, since

1. It is cheaper according to the `.lp` file
2. It's peak capacity of 110 MW_th is sufficient for peak demand of 87.77 MW_th

> If everything is fine with the model, should we decommission the plant (at least 2050)...?

You mean we take it out for NEP and ISE scenarios entirely?
We cann do it, but we don't have to. The only difference it makes, is that fixed costs aren't accounted anymore

**Comment by nesnoj** [comment]
> Does not really make sense to me that it changed with the new scenarios. If GuD Dessau was used in old NEP2035_* scenarios, with lesser RE generation in new scenarios it should be less dispatched.

You're right, that's how I'd expect the system to work. I recall that it was used more but I'm not fully sure..

> 2. It's peak capacity of 110 MW_th is sufficient for peak demand of 87.77 MW_th

Are you talking about the CCGT B-W or Dessau? (CCGT B-W supplies electricity only, no heat)
I queried Dessau with
```
SELECT max(th_hh_efh+th_rca+th_hh_mfh)*0.42
	FROM windnode.wn_abw_demandts
	WHERE ags_id=15001000;
```
(0.42 = district heating share)
which gives 120 MW peak that cannot be fully covered by the CCGT (60 MW). But you're right, the gas boilers' nom. power equals the peak load and is therefore able to fully supply the reqd. heat demand. I checked this and the entire heat really comes from gas boilers - I'm quite curious that they seem to be cheaper than the CCGT. o_O

> You mean we take it out for NEP and ISE scenarios entirely?
> We cann do it, but we don't have to. The only difference it makes, is that fixed costs aren't accounted anymore

Yes, if the plant isn't dispatched at all we should think about decommissioning at least for 2050 (it was just reconditioned lately and will surely be running in 2035) to save costs.

I'd assume that the dispatch won't change too much when setting methane share to 0 (#158) as gas costs are equal für CCGT+gas boilers. But maybe the resulting cheaper el. could lead to more dispatch..
